AUTHORS

```diff
@@ -1,3 +1,6 @@
 Timur Shemsedinov <timur.shemsedinov@gmail.com>
+Dimon Durak <Dimon Durak>
```

Comment:
Thanks, I've added an entry for you in `.mailmap` and regenerated the list of contributors.